docs/webiny-apps/headless-cms/features/access-tokens.md

```diff
@@ -0,0 +1,23 @@
+---
+id: access-tokens
+title: Access Tokens
+sidebar_label: Access Tokens
+---
+
+Access Tokens is another feature of Headless CMS :)
+
+You can find Access tokens in the menu shown in the image below:
+
+![Access tokens in menu](/img/webiny-apps/headless-cms/features/content-modeling/access-tokens-menu.png)
+
+Access tokens provide you access to certain environments.
```

Comment:
Note: This is perhaps a personal preference
Maybe clarify a bit on what you mean by certain environments.
Something like:
By creating an access token, you are able to make a specific environment like `dev` only accessible when using a `dev` 
 access token. (The idea here is maybe provide a soft example)